components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/model/UserRecoveryData.java

```diff
@@ -45,6 +47,13 @@ public UserRecoveryData(User user, String secret, Enum recoveryScenario) {
         this.recoveryScenario = recoveryScenario;
     }
 
+    public UserRecoveryData(User user, String secret, Enum recoveryScenario, Enum recoveryStep, boolean isExpired) {
```

Comment:
Shall we add a new line after the method signature?

Reply:
Addressed following the generic coding style as mentioned.